.github/workflows/ci.yml

```diff
@@ -74,6 +103,15 @@ jobs:
         env:
           NX_AGENT_NAME: ${{ matrix.agent }}
 
+      - name: Revoke and delete GPG key
+        run: |
+          # As instructed in the revocation file, the colon needs to first be removed
+          sed -i "s/:-----BEGIN PGP PUBLIC KEY BLOCK-----/-----BEGIN PGP PUBLIC KEY BLOCK-----/" $REVOCATION_FILE
```

Comment:
```suggestion
          sed -i "s/:-----BEGIN GPG PUBLIC KEY BLOCK-----/-----BEGIN GPG PUBLIC KEY BLOCK-----/" $REVOCATION_FILE
```

Reply:
This one is not a typo. If you were to open a revocation file in a text editor, it would look something like this:

```
This is a revocation certificate for the OpenPGP key:

pub   rsa2048 2023-12-11 [SC]
      81CFAE4BD0034D6D1AD443408F11F369BDA6B541
uid          Tester McPerson <test@example.com>

A revocation certificate is a kind of "kill switch" to publicly
declare that a key shall not anymore be used.  It is not possible
to retract such a revocation certificate once it has been published.

Use it to revoke this key in case of a compromise or loss of
the secret key.  However, if the secret key is still accessible,
it is better to generate a new revocation certificate and give
a reason for the revocation.  For details see the description of
of the gpg command "--generate-revocation" in the GnuPG manual.

To avoid an accidental use of this file, a colon has been inserted
before the 5 dashes below.  Remove this colon with a text editor
before importing and publishing this revocation certificate.

:-----BEGIN PGP PUBLIC KEY BLOCK-----
Comment: This is a revocation certificate

iQE2BCABCAAgFiEEgd+uS9ADTW0a1ENAjxHzab2msUEFAmV2yBoCHQAACgkQjxHz
...

-----END PGP PUBLIC KEY BLOCK-----
```

Reply:
Ah sorry that was me just not being familiar enough with these specs/standards

Reply:
To be honest it's an easy thing to miss - your other suggestions were indeed typos, so it made sense for this to also be a typo :) It's good to double-check in these situations, so your feedback was spot on!